scripts/CEE/cluster-health-check/metadata.yaml

```diff
@@ -0,0 +1,95 @@
+file: run-oc-hc.sh
+name: cluster-health-check
+description: Check the overall health of an OpenShift cluster
+author: Givaldo Lins
+allowedGroups:
+  - CEE
+  - SREP
+  - CSSRE
+  - LPSRE
+  - MTSRE
+rbac:
+  clusterRoleRules:
+    - verbs:
+        - get
+        - list
+        - watch
+      apiGroups:
+        - ''
+      resources:
+        - namespaces
+    - verbs:
+        - get
+        - list
+        - watch
+        - patch
+      apiGroups:
+        - monitoring.coreos.com
+      resources:
+        - alertmanagers
+    - verbs:
+        - get
+        - list
+      apiGroups:
+        - metrics.k8s.io
+      resources:
+        - nodes
+    - verbs:
+        - get
+        - list
+      apiGroups:
+        - machineconfiguration.openshift.io
+      resources:
+        - machineconfigpools
+        - machineconfig
+    - verbs:
+        - get
+        - list
+      apiGroups:
+        - config.openshift.io
+      resources:
+        - clusteroperators
+        - clusterversions
+    - verbs:
+        - get
+        - list
+      apiGroups:
+        - route.openshift.io
+      resources:
+        - routes
+    - verbs:
+        - get
+        - list
+      apiGroups:
+        - ''
+      resources:
+        - nodes
+        - events
+    - verbs:
+        - get
+        - list
+      apiGroups:
+        - certificates.k8s.io
+      resources:
+        - certificatesigningrequests
+    - verbs:
+        - get
+        - watch
+        - list
+        - create
+        - delete
```

Comment:
What is the check doing that it needs access to create and delete pods?

Reply:
It spins up a pod to test DNS resolution and egress and removes it afterwards:
https://github.com/givaldolins/openshift-cluster-health-check/blob/main/oc-hc/cmd/networkStatus.go#L36-L60

Reply:
We have the network verifier to test egress. Allowing this to spin up an arbitrary pod in any namespace is a possible privilege escalation

Reply:
+1, can we try to avoid any CREATE/PATCH/UPDATE operations in the health check itself?
Ideally, we should not also try to avoid execute any command on the cluster as well. But if it is really needed, I will suggest to find existing redhat/SRE managed pod and execute within that rather than create/delete a new one.

Reply:
I have removed the writing permissions and am updating the oc-hc code to not run that check by default